pkgs/applications/networking/mailreaders/betterbird/default.nix

```diff
@@ -110,6 +112,22 @@ in ((buildMozillaMach {
   webrtcSupport = false;
 
   pgoSupport = false; # console.warn: feeds: "downloadFeed: network connection unavailable"
+
+  # Betterbird is broken the same way Thunderbird is regarding VTZONE parsing in
+  # [this issue][1] which is ultimately due to a [bug in Unicode ICU][2].
+  # Therefore, Betterbird should apply the same patch that Thunderbird does at
+  # present (see ../thunderbird/packages.nix, or [permalink][3]).
+  #
+  # [1]: https://bugzilla.mozilla.org/show_bug.cgi?id=1843007
+  # [2]: https://unicode-org.atlassian.net/browse/ICU-22132
+  # [3]: https://github.com/NixOS/nixpkgs/blob/2c7f3c0fb7c08a0814627611d9d7d45ab6d75335/pkgs/applications/networking/mailreaders/thunderbird/packages.nix#L82
```

Comment:
And that line. It is the same git repo which shouldn't be to hard to find again.